src/game/entities/BasicCircle.js

```diff
@@ -21,7 +21,13 @@ class BasicCircle extends GameEntity {
     graphic.width = radius * 2;
     graphic.height = radius * 2;
     graphic.anchor.set(0.5, 0.5);
-    game.app.stage.addChild(graphic);
+
+    // This is also used for graphic creation,
+    //in that case we do not want to spawn graphic to game
+    if(spawn){
+      game.app.stage.addChild(graphic);
+    }
+

```

Comment:
A better solution here is to not make GameEntities add themselves to the stage and instead move that logic to the EntityHandler. This will make creating any GameEntity free from unintended consequences.

Reply:
Agree, that would be nicer. Requires quite some refactoring though, not much related to this pr. To remove the game form enitity  constructor would require a different treatment of the basic resources (circle pictures) as well.

Reply:
Ok, I can fix that and throw it in with dangerbot in a bit.